<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automação 2.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.ico</normaloff>../img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject{
	background-color: black;
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QFrame" name="dados">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>452</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
color: white;
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lblUser">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>150</y>
            <width>130</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Sitka Heading</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Undefined</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>40</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>5</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../img/icon.ico</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="metaLbl">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>180</y>
            <width>191</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Semibold</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Meta: R$ 0000000</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="logout">
            <property name="font">
             <font>
              <family>Segoe MDL2 Assets</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);
	color: white;
	border: 2px solid red;
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>LOGOUT</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/del.png</normaloff>../img/del.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>35</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+Del</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="main">
      <item>
       <widget class="QFrame" name="buttons">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_3">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="btnhxh">
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 0;</string>
               </property>
               <property name="text">
                <string>Hora-Hora</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/hxh.png</normaloff>../img/hxh.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="shortcut">
                <string>F5</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnestore">
               <property name="font">
                <font>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 0;</string>
               </property>
               <property name="text">
                <string>E-Store</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/cart.png</normaloff>../img/cart.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="shortcut">
                <string>F12</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="paBtn">
               <property name="font">
                <font>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 0;</string>
               </property>
               <property name="text">
                <string>P.A</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/pa.png</normaloff>../img/pa.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnvj">
               <property name="font">
                <font>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 0;</string>
               </property>
               <property name="text">
                <string>V.J</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/vj.png</normaloff>../img/vj.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_4">
            <layout class="QVBoxLayout" name="verticalLayout_6"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="corp">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>364</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(51, 51, 51);
color: white;
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 18px;</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>820</width>
              <height>418</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Texto</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPlainTextEdit" name="entex">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
               </property>
               <property name="placeholderText">
                <string>Cole o texto aqui (Ctrl + V)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>820</width>
              <height>418</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Metas</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QPushButton" name="atualizarEstore">
                 <property name="font">
                  <font>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Atualizar Estore</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../img/pw.png</normaloff>../img/pw.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="metaBtn">
                 <property name="font">
                  <font>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Atualizar Metas</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../img/excel.png</normaloff>../img/excel.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>820</width>
              <height>418</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Preferencias</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QPushButton" name="btnEdit">
                 <property name="text">
                  <string>EDITAR</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../img/edit.png</normaloff>../img/edit.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="btnAdd">
                 <property name="font">
                  <font>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>ADICIONAR</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../img/user.png</normaloff>../img/user.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="btnAtua">
                 <property name="font">
                  <font>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>ATUALIZAR NOME</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../img/update.png</normaloff>../img/update.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btnSenha">
                 <property name="font">
                  <font>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>TROCAR SENHA</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../img/key.png</normaloff>../img/key.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
